app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -68,6 +68,15 @@ export default function RegistrationAdministrationList({ competitionInfo }) {
 
   const actionsRef = useRef();
 
+  const pendingRef = useRef();
+  const waitlistRef = useRef();
+  const approvedRef = useRef();
+  const cancelledRef = useRef();
+  const rejectedRef = useRef();
+  const tableRefs = useMemo(() => ({
+    pendingRef, waitlistRef, approvedRef, cancelledRef, rejectedRef,
+  }), []);
```

Comment:
I like the concise notation, but I am a bit afraid it would make the code "vulnerable" to refactors. What about using the actual `competition_status` enum values (like `"pending"`, `"approved"`, etc) as keys?
That would make it a bit more verbose here but more standardized and easier to maintain in the future